R/mail.R

```diff
@@ -21,11 +21,11 @@
 #' @param mailing_list The name of the mailing list being downloaded (e.g. "https://mta.openssl.org/pipermail/openssl-announce/")
 #' @param start_year_month The year and month of the first file to be downloaded (format: 'YYYYMM')
 #' @param end_year_month The year and month of the last file to be downloaded (format: 'YYYYMM', or use 'format(Sys.Date(), "%Y%m")' for the current month)
-#' @param save_folder_path The folder path in which all the downloaded pipermail files will be stored
+#' @param mbox The folder path in which all the downloaded pipermail files will be stored
```

Comment:
I believe you want to keep at least `folder_path` here. The `mbox` name does not make it clear to me a path to anything is required, nor a folder. Unclear why the configs are being changed either, it is breaking the consistency.